gems/sorbet-runtime/lib/types/helpers.rb

```diff
@@ -16,6 +16,10 @@ def interface!
     Private::Abstract::Declare.declare_abstract(self, type: :interface)
   end
 
+  def final!
```

Comment:
Can we only keep the `final` that's related to methods in this PR?

Reply:
Yes.